src/ds/d400/d400-auto-calibration.cpp

```diff
@@ -61,57 +47,15 @@ namespace librealsense
         uint16_t tableSize;  // 512 bytes
     };
 
-    struct DscResultParams
-    {
-        uint16_t m_status;
-        float    m_healthCheck;
-    };
-
     struct DscResultBuffer
     {
-        uint16_t m_paramSize;
-        DscResultParams m_dscResultParams;
-        uint16_t m_tableSize;
-    };
-
-    enum rs2_dsc_status : uint16_t
-    {
-        RS2_DSC_STATUS_SUCCESS = 0, /**< Self calibration succeeded*/
-        RS2_DSC_STATUS_RESULT_NOT_READY = 1, /**< Self calibration result is not ready yet*/
-        RS2_DSC_STATUS_FILL_FACTOR_TOO_LOW = 2, /**< There are too little textures in the scene*/
-        RS2_DSC_STATUS_EDGE_TOO_CLOSE = 3, /**< Self calibration range is too small*/
-        RS2_DSC_STATUS_NOT_CONVERGE = 4, /**< For tare calibration only*/
-        RS2_DSC_STATUS_BURN_SUCCESS = 5,
-        RS2_DSC_STATUS_BURN_ERROR = 6,
-        RS2_DSC_STATUS_NO_DEPTH_AVERAGE = 7
+        uint16_t paramSize;
```

Comment:
I see this struct was changed, won't it impact D400 devices?

Reply:
This struct definition is redundant I will remove it.
I merged the 2 fields of `DscResultParams` into `DscResultBuffer` and moved it to the common. This was the only place `DscResultParams` was used and I think this is simpler and helps readability. D400 flow calls the correct field. The name have changed anyway because it's in ds_calib_common namespace.